Dockerfile

```diff
@@ -18,6 +18,8 @@ EXPOSE 3000 4000
 
 WORKDIR /mastodon
 
+COPY Gemfile Gemfile.lock package.json yarn.lock .yarnclean /mastodon/
```

Comment:
This means you have to reinstall the entire OS inside the container every time any of the source files change.